src/puppetlabs/puppetdb/utils.clj

```diff
@@ -257,3 +257,10 @@
       {:href (to-href data)}
       (-> (sutils/parse-db-json data)
           (update :href to-href)))))
+
+(defn hsql?
+  "given the globals configuration bundle, determine if hsqldb is being used."
+  [globals]
```

Comment:
Usually the pattern I've seen in other projects for this kind of functions is `db-spec` here. I'd prefer we'd do something like that.